source/documentation/guides/api_standards.html.md.erb

```diff
@@ -43,7 +43,17 @@ For further information, see the [OPG Data versioning strategy](https://github.c
 1. Use the [OpenAPI Specification](https://swagger.io/specification/) for documenting APIs
 1. Store in the repo at `/docs/openapi/openapi.yml`
 
-## Security
+## Authentication
+
+1. **Use JSON Web Tokens** (JWTs) to authenticate requests between OPG services
+    1. JWTs should be asymmetrically signed
```

Comment:
Should keys be unique per-API or per-integration? Look at best practices from other organisations